metadetect/lsst_measure.py

```diff
@@ -76,50 +129,74 @@ def measure_weighted_moments(
 
             ormask = ormasks[i]
 
-            # This will insert a single source into the image
-            replacer.insertSource(source.getId())
+            if use_deblended_stamps:
+                # This will insert a single source into the image
+                replacer.insertSource(source.getId())
 
-            meas_task.callMeasure(source, exposure)
+            if meas_task is not None:
+                meas_task.callMeasure(source, exposure)
```

Comment:
Where do the results of this `meas_task` go?

Reply:
They are stored in the source

Reply:
I added a comment